javascript/src/ycell.ts

```diff
@@ -895,6 +903,16 @@ export class YCodeCell
   protected getChanges(events: Y.YEvent<any>[]): Partial<CellChange> {
     const changes = super.getChanges(events);
 
+    const streamOutputEvent = events.find(
+      event =>
+        event.path.length === 3 &&
+        event.path[0] === 'outputs' &&
+        event.path[2] === 'text'
```

Comment:
It's hard to understand this without some concrete example. I guess a comment or a test could help here.

Reply:
The event path is an array leading to the changed property, so to access the `text` property of a cell's stream output, one would do `cell['outputs'][output_idx]['text']` and the event path would then be `['outputs', output_idx, 'text']`. I'll add a comment :+1: